src/utils/utils-vega.js

```diff
@@ -156,7 +157,12 @@ export function createVegaAttrMixin(
         }
         if (capAttrObj.type === "ordinal") {
           ordScale.domain(domainVals).range(capAttrObj.range)
-          rtnVal = ordScale(input)
+          // if color range is not in domain, it's an Other item
+          rtnVal = domainVals.indexOf(input) === -1 ? ordScale("Other") : ordScale(input)
```

Comment:
It's probably safe to count on the "Other" item to be in the range list anytime you have an input that is not in it as it means that the "Other" feature is toggled on in the color palette widget. Anyway if for any reason it's not there, it will default to the first item from the domain as it was doing before.